app/assets/stylesheets/application.tailwind.css

```diff
@@ -1,3 +1,14 @@
-/* @tailwind base; */ /* base disabled because the resets break the current site */ 
+@config "../../../config/tailwind.config.js";
+@tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  @font-face {
+    font-family: "Titillium Web";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(/fonts/Roboto.woff2) format('woff2');
```

Comment:
Yes the font here is wrong but it doesn't affect rendering. I'll update it when we actually merge the design. (I couldn't find the best way to do this without a src).